domain/dcsa_ovs_domain.yaml

```diff
@@ -0,0 +1,254 @@
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/scheduleID'
+        - type: object
+          properties:
+            carrierServiceCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/carrierServiceCode'
+        - type: object
+          properties:
+            vesselIMONumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/vesselIMONumber'
+        - type: object
+          properties:
+            vesselName:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/vesselName'
+        - type: object
+          properties:
+            carrierVoyageNumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/carrierVoyageNumber'
+        - type: object
+          properties:
+            UNLocationCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationCode'
+        - type: object
+          properties:
+            UNLocationName:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationName'
+        - type: object
+          properties:
+            transportCallNumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/transportCallNumber'
+        - type: object
+          properties:
+            facilityTypeCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/facilityTypeCode'
+        - type: object
+          properties:
+            facilityCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/facilityCode'
+        - type: object
+          properties:
+            otherFacility:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/otherFacility'
+      required:
+        - transportCallID
+        - scheduleID
+        - carrierVoyageNumber
+        - UNLocationCode
+        - facilityTypeCode
+        - facilityCode
+    
+    ######################################
+    # Transport Event Related Objects
+    ######################################
+
+    transportEvent:
+      description:  A list of all the transport-events involved in a portCall.
+      type: object
+      allOf:
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_NOTIFICATION_DOMAIN/1.0.0#/components/schemas/transportNotification'
+        - type: object
+          properties:
+            locationType:
+              $ref: '#/components/schemas/locationType'
+        - type: object
+          properties:
+            locationID:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/locationID'
```

Comment:
I think you should update this in DCSA_NOTIFICATION as it seems to be part of the TransportNotification. Keep in mind that NOTIFICATION_DOMAIN is currently **only** used by TNT v3 (which is not yet published) - it is a suggestion that will be presented to the SME's next tuesday...

Reply:
If you do update the TransportNotification in NOTIFICATION_DCSA - then transportEvent might not be needed in this file